handel/src/services/dynamodb/index.ts

```diff
@@ -239,81 +246,16 @@ async function getCompiledDynamoTemplate(stackName: string, ownServiceContext: S
     return handlebars.compileTemplate(`${__dirname}/dynamodb-template.yml`, handlebarsParams);
 }
 
-const TABLE_NAME_ALLOWED_PATTERN = /^[a-zA-Z0-9_\-.]{3,255}$/;
-
 /**
  * Service Deployer Contract Methods
  * See https://github.com/byu-oit-appdev/handel/wiki/Creating-a-New-Service-Deployer#service-deployer-contract
  *   for contract method documentation
  */
 
 export function check(serviceContext: ServiceContext<DynamoDBConfig>, dependenciesServiceContexts: Array<ServiceContext<ServiceConfig>>): string[] {
-    const errors = [];
-    const params = serviceContext.params;
-
-    if (params.table_name && !TABLE_NAME_ALLOWED_PATTERN.test(params.table_name)) {
-        errors.push(`${SERVICE_NAME} - The table_name parameter must be between 3 and 255 characters long and may only include alphanumeric characters, underscores (_), hyphens (-), and dots (.)`);
-    }
-
-    if (!params.partition_key) {
-        errors.push(`${SERVICE_NAME} - The 'partition_key' section is required`);
-    }
-    else {
-        if (!params.partition_key.name) {
-            errors.push(`${SERVICE_NAME} - The 'name' field in the 'partition_key' section is required`);
-        }
-        if (!params.partition_key.type) {
-            errors.push(`${SERVICE_NAME} - The 'type' field in the 'partition_key' section is required`);
-        }
-    }
-
-    // Check throughput
-    errors.push(...autoscaling.checkProvisionedThroughput(params.provisioned_throughput, `${SERVICE_NAME} - `));
```

Comment:
It looks like you're removing this line but I don't see a corresponding removal of the method itself in the autoscaling module. We should probably look at this method and decide whether to keep it. If we keep it we can put this line back in or if we don't need it then we can delete the method.